app/services/claim_service.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+require_dependency 'evss/claims_service'
+
+class ClaimService
+  EVSS_CLAIM_KEYS = %w(openClaims historicalClaims).freeze
+
+  def initialize(user)
+    @user = user
+  end
+
+  def all
+    raw_claims = client.all_claims.body
+    EVSS_CLAIM_KEYS.each_with_object([]) do |key, claims|
+      next unless raw_claims[key]
+      claims << raw_claims[key].map do |raw_claim|
+        find_or_initialize_claim(raw_claim)
+      end
+    end.flatten
+  rescue Faraday::Error::TimeoutError, Timeout::Error
+    Claim.where("data->>'participant_id' = ?", @user.participant_id)
+  end
+
+  def find_by_evss_id(id)
+    raw_claim = client.find_claim_by_id(id).body.fetch('claim', {})
+    find_or_initialize_claim(raw_claim)
+  rescue Faraday::Error::TimeoutError, Timeout::Error
```

Comment:
As an example, you could use the Common::Errors::ClientResponseError in lib folder, and then come up with a rescue_from that could translate those to json api spec... but thats something I was looking to tackle at some point soon.